data/beaconrestapi/src/main/java/tech/pegasys/teku/beaconrestapi/handlers/v1/beacon/StateValidatorsUtil.java

```diff
@@ -13,27 +13,17 @@
 
 package tech.pegasys.teku.beaconrestapi.handlers.v1.beacon;
 
-import static tech.pegasys.teku.infrastructure.http.RestApiConstants.STATUS;
-
 import java.util.List;
-import java.util.Map;
 import java.util.Set;
 import java.util.stream.Collectors;
 import tech.pegasys.teku.api.exceptions.BadRequestException;
 import tech.pegasys.teku.api.response.v1.beacon.ValidatorStatus;
-import tech.pegasys.teku.infrastructure.restapi.endpoints.ListQueryParameterUtils;
 
 public class StateValidatorsUtil {
 
-  public Set<ValidatorStatus> parseStatusFilter(final Map<String, List<String>> queryParameters) {
-    if (!queryParameters.containsKey(STATUS)) {
-      return Set.of();
-    }
-
+  public Set<ValidatorStatus> parseStatusFilter(final List<String> statusList) {
     try {
```

Comment:
this may be redundant if we can get the type def to produce the correct type for us, which i think we can...

Reply:
The type definition already uses an enum type for `ValidatorStatus`. Does that mean this isn't needed?